Jenkinsfile

```diff
@@ -20,7 +21,7 @@ pipeline {
         POMFIX = 'libs/libs-pomfix,libs/libs-pomfix-run'
         LARGE_MVN_OPTS = '-Xmx8192M -Xss128M -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC '
         LINUX_MVN_RANDOM = '-Djava.security.egd=file:/dev/./urandom'
-        COVERAGE_EXCLUSIONS = '**/test/**,**/itests/**,**/*Test*,**/sdk/**,**/*.js,**/node_modules/**,**/jaxb/,**/wsdl/,**/nces/sws/**,**/*.adoc,**/*.txt,**/*.xml'
+        COVERAGE_EXCLUSIONS = '**/test/**/*,**/itests/**/*,**/*Test*,**/sdk/**/*,**/*.js,**/node_modules/**/*,**/jaxb/**/*,**/wsdl/**/*,**/nces/sws/**/*,**/*.adoc,**/*.txt,**/*.xml,**/platform-solr-server-standalone/**/*'
```

Comment:
❓ Platform-solr-server-standalone doesn't have any code donated to Codice, correct?

Reply:
@clockard was there something being analyzed in `platform-solr-server-standalone`? It looks like most of the files in this war would have been ignored based on the existing exclusions.

Reply:
I added that the same time I updated the rest of the path expressions so it might not be doing anything. I just wanted to make sure that nothing from that module was included in the results but didn't look at what was actually in that module.